_tests/browser_tests/jenkins_conf.js

```diff
@@ -0,0 +1,41 @@
+var env = require('./environment.js');
```

Comment:
Ideally the variable name should be [consistent with the filename](https://github.com/cfpb/front-end/blob/master/javascript-modules-commonjs.md#module-naming).